client/results.html

```diff
@@ -3,12 +3,13 @@
 
 <head>
```

Comment:
It would maybe be a good idea to use just one results page, and use the for loop in javascript to display multiple elements programatically